.svgo.yml

```diff
@@ -19,7 +19,11 @@ plugins:
   - minifyStyles: true
   - moveElemsAttrsToGroup: true
   - moveGroupAttrsToElems: true
-  - removeAttrs: false
+  - removeAttrs:
+      attrs: 'fill'
+  - addAttributesToSVGElement:
+      attributes:
+        - fill: 'currentColor'
```

Comment:
Can we solve this with `svg { @apply fill-current; }` instead?

Reply:
Ah! It’s not entirely the same in our setup. Because we're wrapping the svg markup in span, the svg itself never gets the attribute value in some situations and for icons we near always want the currentColor on fill. I borrowed this approach from Tailwind's own SVG library: https://github.com/tailwindlabs/heroicons/blob/master/svgo.solid.yaml

Secondarily, this value is how the Tailwind SVG Plugin I wrote works. When a file is scanned and currentColor exists on fill or stroke, you replace it with the color vars you apply in the plugin.